Sources/UIFont.swift

```diff
@@ -174,16 +174,19 @@ extension NSAttributedString {
 
 extension String {
     public func size(with font: UIFont, wrapLength: CGFloat = 0) -> CGSize {
-        guard let renderer = font.renderer else { return .zero }
+        guard
+            let renderer = font.renderer,
+            let screen = UIScreen.main
```

Comment:
Interesting, I thought that no rendering happens at all when `UIScreen.main` does not exist. AFAIK this code gets executed on every render and therefore I thought it looks better if we render a few frames with `.zero` size and then render with correct sizing when `UIScreen.main` was (hopefully) recreated in the meantime.

Something like this would still fix the crash:
```swift
let screenScale = UIScreen.main?.scale ?? 2

...
```

Reply:
The thing is that this code fortunately (for performance reasons) / unfortunately (for visual fidelity reasons) does not get called every frame. So I think either way we have an issue if this code is being called when UIScreen.main does not exist. So IMO the issue actually lies elsewhere – by fixing the crash here we're just pushing it down the line somewhere else. In reality we just shouldn't be rendering or laying out anything if UIScreen doesn't exist. Maybe there's a way to solve that issue more fundamentally rather than prevent the crash deep in the code here